moto/__init__.py

```diff
@@ -235,6 +235,11 @@ def f() -> "xray_segment_type":
 mock_xray = lazy_load(".xray", "mock_xray")
 mock_xray_client = lazy_load(".xray", "mock_xray_client")
 mock_wafv2 = lazy_load(".wafv2", "mock_wafv2")
+mock_backup = lazy_load(".backup", "mock_backup", boto3_name="backup")
+mock_backup = lazy_load(".backup", "mock_backup", boto3_name="backup")
+mock_backup = lazy_load(".backup", "mock_backup", boto3_name="backup")
+mock_backup = lazy_load(".backup", "mock_backup", boto3_name="backup")
+mock_backup = lazy_load(".backup", "mock_backup", boto3_name="backup")
```

Comment:
This needs to only be added once (I know it's a bug in the `scaffold`-script, it adds this line without checking)